worlds/rabi_ribi/regions.py

```diff
@@ -180,7 +180,8 @@ def set_regions(self):
         if not self.options.include_plurkwood and not ut_helpers.should_regenerate_seed_for_universal_tracker(self.world):
             self.unreachable_regions.update(plurkwood_regions)
 
-        if not self.options.include_warp_destination:
+        if not self.options.include_warp_destination and not self.options.include_post_game:
+            #TODO Warn users that warp destination is included, OR split the regions from the locations
```

Comment:
There's no need for a warning about this; the locations are already separated from the regions.